js/main.js

```diff
@@ -8,6 +12,10 @@ firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);
 //    after the API code downloads.
 var player;
 function onYouTubeIframeAPIReady () {
+  window.addEventListener('resize', function() {
+    setTimeout(resizePlayer, 15);
+  });
+  resizePlayer();
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ["resizePlayer" was used before it was defined](https://www.codacy.com/app/j-hasseriis/cepheus/file/1860405879/issues/source?bid=3085146&fileBranchId=3085654#l18)